src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -24,12 +24,12 @@
 [assembly: InternalsVisibleTo("OrleansRuntime")]
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansAWSUtils")]
 [assembly: InternalsVisibleTo("OrleansManager")]
 [assembly: InternalsVisibleTo("LoadTestGrains")]
 [assembly: InternalsVisibleTo("UnitTests")]
 [assembly: InternalsVisibleTo("UnitTestGrainInterfaces")]
 [assembly: InternalsVisibleTo("UnitTestGrains")]
 [assembly: InternalsVisibleTo("TesterInternal")]
 [assembly: InternalsVisibleTo("TestInternalGrainInterfaces")]
-[assembly: InternalsVisibleTo("TestInternalGrains")]
-
+[assembly: InternalsVisibleTo("TestInternalGrains")]
```

Comment:
When added `[assembly: InternalsVisibleTo("OrleansAWSUtils")]` just removed an empty line. Not that it worth...